interface/src/app/gene/network/network.js

```diff
@@ -235,16 +255,29 @@ angular.module('adage.gene.network', [
           .call(edgeTip);
 
         geneTip.html(getGeneInfo);
-        network.onGene('mouseover.custom', geneTip.show);
-        network.onGene('mouseout.custom', geneTip.hide);
-        network.onEdge('mouseover.custom', showEdgeTip);
-        network.onEdge('mouseout.custom', hideEdgeTip);
+        network.onGene('click.custom', geneTip.show);
+        network.onEdge('click.custom', showEdgeTip);
```

Comment:
I added one line of text `Click the gene or edge to see more details` above the network graph, but that seems a little distracting. I would like to reply on the upcoming **Help** tab to describe the usage of each module.